torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -1493,14 +1493,18 @@ at::Tensor interpolate(
 
     if (warn_recompute_scale_factor) {
       TORCH_WARN(
-          "The default behavior for interpolate/upsample with float scale_factor will change "
```

Comment:
This op is no longer generated, so it shouldn't be changed bc it will affect serialized models.